server/api/prospect_handler.py

```diff
@@ -52,25 +50,26 @@ def upload_prospects():
     # Add to db
     prospect_json = prospect_json.copy()
     prospect_list = list(prospect_json['prospects'].values())
-    owner_email = prospect_json['owner']
-    owner = User.get_by_email(owner_email)
+    owner = User.get_by_email(get_jwt_identity()["email"])
     dup_prospects = 0
```

Comment:
The dup_prospects is not used

Reply:
Should be fixed in latest commit. Also removed from frontend as well